components/Account.php

```diff
@@ -366,18 +376,26 @@ public function onUpdate()
         if (!$user = $this->user()) {
             return;
         }
+        
+        $data = post();
+        
+        if ($this->updateRequiresPassword()) {
+            if (!$user->checkHashValue('password', $data['password_old'])) {
+                throw new ValidationException(['password' => Lang::get('rainlab.user::lang.account.invalid_deactivation_pass')]);
```

Comment:
@cbacon93 Would you mind changing this to use its own Lang file reference so people can customise the message if they wish?